src/main/java/it/gov/pagopa/pu/debtpositions/service/create/debtposition/CreateDebtPositionServiceImpl.java

```diff
@@ -0,0 +1,93 @@
+  private final ValidateDebtPositionService validateDebtPositionService;
+  private final DebtPositionService debtPositionService;
+  private final GenerateIuvService generateIuvService;
+  private final InstallmentNoPIIRepository installmentNoPIIRepository;
+
+  public CreateDebtPositionServiceImpl(AuthorizeOperatorOnDebtPositionTypeService authorizeOperatorOnDebtPositionTypeService,
+                                       DebtPositionMapper debtPositionMapper,
+                                       ValidateDebtPositionService validateDebtPositionService,
+                                       DebtPositionService debtPositionService, GenerateIuvService generateIuvService,
+                                       InstallmentNoPIIRepository installmentNoPIIRepository) {
+    this.authorizeOperatorOnDebtPositionTypeService = authorizeOperatorOnDebtPositionTypeService;
+    this.debtPositionMapper = debtPositionMapper;
+    this.validateDebtPositionService = validateDebtPositionService;
+    this.debtPositionService = debtPositionService;
+    this.generateIuvService = generateIuvService;
+    this.installmentNoPIIRepository = installmentNoPIIRepository;
+  }
+
+  @Override
+  public DebtPositionDTO createDebtPosition(DebtPositionDTO debtPositionDTO, Boolean massive, Boolean pagopaPayment) {
+    log.info("START Create DebtPosition...");
+    String accessToken = SecurityUtils.getAccessToken();
+
+    verifyAuthorization(debtPositionDTO);
+    validateInstallments(debtPositionDTO, accessToken);
+
+    DebtPosition debtPosition = debtPositionMapper.mapToModel(debtPositionDTO).getFirst();
+
+    debtPosition.getPaymentOptions().stream()
+      .flatMap(po -> po.getInstallments().stream())
+      .forEach(installmentNoPII -> {
+        long countDuplicates = installmentNoPIIRepository.countExistingDebtPosition(debtPosition.getOrganizationId(), installmentNoPII.getIud(), installmentNoPII.getIuv(), installmentNoPII.getNav());
+        if (countDuplicates > 0) {
+          log.error("Duplicate record found for Installment with id {}", installmentNoPII.getInstallmentId());
+          throw new ConflictErrorException("Duplicate records found: the provided data conflicts with existing records.");
+        }
+      });
+
+    if (Boolean.TRUE.equals(pagopaPayment)) {
+      debtPositionDTO.getPaymentOptions().stream()
+        .flatMap(po -> po.getInstallments().stream())
+        .forEach(installment -> {
+          String orgFiscalCode = installment.getDebtor().getFiscalCode();
+          String generatedIuv = generateIuvService.generateIuv(orgFiscalCode, accessToken);
+          installment.iuv(generatedIuv);
+        });
+    }
+
+    DebtPositionDTO savedDebtPosition = debtPositionService.saveDebtPosition(debtPositionDTO);
+
+    log.info("END Create DebtPosition...");
+    return savedDebtPosition;
+  }
+
+  private void verifyAuthorization(DebtPositionDTO debtPositionDTO) {
+    log.info("START Verify Authorization...");
```

Comment:
too many logs for INFO level
```suggestion
    log.debug("START Verify Authorization...");
```